### DISCUSSION TITLE
ReferenceError: Cannot access '__WEBPACK_DEFAULT_EXPORT__' before initialization

### DISCUSSION THREAD

**Discussion by mvrgopaal** [webpack/webpack#18970] [Q&A]
# Bug report

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Other relevant information:**
webpack version:  
Node.js version:
Operating System:
Additional tools:

**Comment by alexander-akait** [comment]
> If the current behavior is a bug, please provide the steps to reproduce.

Please fill in

**Reply by mvrgopaal** [reply]
Hello Alexander,  Thank you for creating this discussion.  We are upgrading our webpack from 4.x to 5.x and we have fixed all the known issues and the build is successful. But the application is not coming up and throwing this error.  Any insights on what could be the issue.

**Reply by alexander-akait** [reply]
Most likely you have a circular import, try it - https://github.com/aackerman/circular-dependency-plugin, In some complex cases it is difficult to implement so you may get this error, we have plans to improve the error output, but it is not implemented yet

**Reply by mvrgopaal** [reply]
Thank you Alexander for the help. We are seeing the following errors but these are all from node_modules itself not from our code.
[INFO] ERROR in node_modules/d3-interpolate/src/array.js -> node_modules/d3-interpolate/src/value.js -> node_modules/d3-interpolate/src/array.js
[INFO] ERROR in node_modules/d3-interpolate/src/object.js -> node_modules/d3-interpolate/src/value.js -> node_modules/d3-interpolate/src/object.js
[INFO] ERROR in node_modules/d3-interpolate/src/value.js -> node_modules/d3-interpolate/src/array.js -> node_modules/d3-interpolate/src/value.js
[INFO] ERROR in node_modules/recharts/es6/util/ChartUtils.js -> node_modules/recharts/es6/util/getLegendProps.js -> node_modules/recharts/es6/util/ChartUtils.js
[INFO] ERROR in node_modules/recharts/es6/util/getLegendProps.js -> node_modules/recharts/es6/util/ChartUtils.js -> node_modules/recharts/es6/util/getLegendProps.js

**Reply by alexander-akait** [reply]
Could you create a reproducible repository and I will look at possible solutions?
